independent-publisher-connectors/SOS Inventory/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2218 @@
+										"example": "A62"
+									}
+								}
+							},
+							"workcenter": {
+								"type": "object",
+								"properties": {
+									"id": {
+										"type": "integer",
+										"description": "Pick Ticket workcenter ID",
+										"example": "10"
+									},
+									"name": {
+										"type": "string",
+										"description": "Pick ticket Workcenter Name",
+										"example": "Punch"
+									}
+								}
+							},
+							"tax": {
+								"type": "object",
+								"properties": {
+									"taxable": {
+										"type": "boolean"
+									},
+									"taxCode": {
+										"type": "object",
+										"properties": {
+											"id": {
+												"type": "integer",
+												"description": "Pick Ticket TaxCode ID",
+												"example": "1"
+											},
+											"name": {
+												"type": "string",
+												"description": "Pick ticket Taxcode Name",
+												"example": "TAX"
+											}
+										}
+									}
+								}
+							},
+							"description": {
+								"type": "string",
+								"description": "Line Item Description"
+							},
+							"quantity": {
+								"type": "number",
+								"description": "Line Item Quantity",
+								"example": "1"
+							},
+							"weight": {
+								"type": "number",
+								"description": "Line Item weight",
+								"example": "12"
+							},
+							"volume": {
+								"type": "number",
+								"description": "Line Item Volume",
+								"example": "12"
+							},
+							"weightunit": {
+								"type": "string",
+								"example": "lb"
+							},
+							"volumeunit": {
+								"type": "string",
+								"example": "cbm"
+							},
+							"uom": {
+								"type": "object",
+								"properties": {
+									"id": {
+										"type": "integer",
+										"description": "UOM ID",
+										"example": "10"
+									},
+									"name": {
+										"type": "string",
+										"description": "UOM Name",
+										"example": "case"
+									}
+								}
+							},
+							"lot": {
+								"type": "object",
+								"properties": {
+									"id": {
+										"type": "integer",
+										"description": "Lot ID",
+										"example": "10"
+									},
+									"name": {
+										"type": "string",
+										"description": "Lot Name",
+										"example": "2023-03-15"
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		},
+		"customer": {
+			"type": "object",
+			"properties": {
+				"id": {
+					"type": "number",
+					"description": "customer ID",
+					"example": "123"
+				},
+				"starred": {
+					"type": "integer",
+					"example": "0"
+				},
+				"syncToken": {
+					"type": "integer",
+					"description": "Customer syncToken",
+					"example": "0"
+				},
+				"name": {
+					"type": "string",
+					"description": "Customer Name",
+					"example": "ABC Plastics"
+				},
+				"fullname": {
+					"type": "string",
+					"description": "Customer fullname",
+					"example": ""
```

Comment:
@Harold-Anderson empty values like this one are not allowed and can be removed unless required.